site/content/concepts/resource-limits.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Resource limits"
+weight = 196
++++
+
+Eclipse Hono™ users can set the resource limits such as the maximum number of device connections allowed per tenant or the allowed data volume of the messages over a period of time per tenant. 
+
+Hono specifies an API `ResourceLimitChecks` that is used by the protocol adapters for the verification of the configured resource-limits. A default implementation of this API is shipped with Hono. This default implementation uses the metrics data from the Prometheus back end to verify the resource-limits if any configured. To enable and use the default Prometheus based limits implementation, please refer to the protocol adapter admin guides (configuration parameters starting with `hono.plan.prometheusBased`). If needed, Hono users can bring their own implementation of the above API and use it. The resource-limits for a tenant can be set using the tenant configuration. Please refer to the [Tenant API]({{< relref "/api/Tenant-API.md#request-payload" >}}) for more details.
```

Comment:
IMHO we should not explicitly mention the configuration parameter names. Referring the reader to the admin guide should be sufficient and prevents duplication of this information which we will have a hard time keeping in sync ...